src/lib/ledgers.js

```diff
@@ -2,20 +2,29 @@
 
 const _ = require('lodash')
 const co = require('co')
+const EventEmitter = require('eventemitter2')
 const PluginStore = require('../lib/pluginStore.js')
-const ilpCore = require('ilp-core')
 const TradingPairs = require('./trading-pairs')
 const logger = require('../common/log')
 const log = logger.create('ledgers')
 
-class Ledgers {
+class Ledgers extends EventEmitter {
   constructor ({ config, routingTables }) {
+    super()
+    this.pluginList = [] // LedgerPlugin[]
+    this.plugins = {} // { prefix ⇒ LedgerPlugin }
     this.tables = routingTables
     this._config = config
-    this._core = new ilpCore.Core({ routingTables })
     this._pairs = new TradingPairs()
     this._ledgers = new Map()
     this.requestHandler = co.wrap(this._requestHandler.bind(this))
+
+    const ledgers = this
+    this._relayEvent = function () {
+      const event = arguments[0]
+      const args = Array.prototype.slice.call(arguments, 1)
+      return ledgers.emitAsync.apply(ledgers, [event, this].concat(args))
```

Comment:
`emitAsync` is supported in EventEmitter2 but not EventEmitter. Is there any other way we can do this, or should we require plugins to expose the EventEmitter2 interface rather than the core node one?